lib/ObjWriter/.nuget/runtime.json

```diff
@@ -2,17 +2,17 @@
   "runtimes": {
     "win7-x64": {
       "Microsoft.DotNet.ObjectWriter": {
-        "toolchain.win7-x64.Microsoft.DotNet.ObjectWriter": "1.0.12-prerelease-00001"
+        "toolchain.win7-x64.Microsoft.DotNet.ObjectWriter": "1.0.13-prerelease-00001"
```

Comment:
There is the corresponding file `toolchain.win7-x64.Microsoft.DotNet.ObjectWriter.nuspec` which also needs to be updated. Same as other platforms.